quinn-proto/src/connection.rs

```diff
@@ -2598,7 +2603,10 @@ where
     ///
     /// This does not ensure delivery of outstanding data. It is the application's responsibility to
     /// call this only when all important communications have been completed, e.g. by calling
-    /// `finish` on outstanding streams and waiting for the corresponding `StreamFinished` event.
+    /// [`Connection::finish`] on outstanding streams and waiting for the corresponding
+    /// `StreamFinished` event.
+    ///
+    /// It is safe to call this if [`Connection::outbound_streams`] returns 0.
```

Comment:
This should be reworded as "safe" has a different very specific meaning in rust.

Reply:
Done.